daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -104,17 +105,38 @@ The diagram below shows sequence 1-7 again on a local machine showing the API ca
+4. The Node.js app's sidecar forwards the request to the Node.js app. The Node.js app performs its business logic, logging the incoming message and then persist the order ID into Redis (not shown in the diagram).
 5. The Node.js app sends a response to the Python app through the Node.js sidecar.
-6. Dapr forwards the response to the Python Dapr sidecar
+6. Dapr forwards the response to the Python Dapr sidecar.
 7. The Python app receives the response.
 
-## Next steps
+## Try out service invocation 
+### Quickstarts & tutorials
+The Dapr docs contain multiple quickstarts that leverage the service invocation building block in different example architectures. To get a straight-forward understanding of the service invocation api and it's features we recommend starting with our quickstarts: 
+
+| Quickstart/tutorial | Description |
+| ------------------- | ----------- |
+| [Service invocation quickstart]({{< ref serviceinvocation-quickstart.md >}}) | This quickstart gets you interacting directly with the service invocation building block. |
+| [Hello world tutorial](https://github.com/dapr/quickstarts/blob/master/tutorials/hello-world/README.md) | This tutorial shows how to use both the service invocation and state management building blocks all running locally on your machine. |
+| [Hello world kubernetes tutorial](https://github.com/dapr/quickstarts/blob/master/tutorials/hello-kubernetes/README.md) | This tutorial walks through using Dapr in kubernetes and covers both the service invocation and state management building blocks as well. |
+
 
-- Follow these guides on:
-  - [How-to: Invoke services using HTTP]({{< ref howto-invoke-discover-services.md >}})
-  - [How-To: Configure Dapr to use gRPC]({{< ref grpc >}})
-  - [How-to: Invoke services using gRPC]({{< ref howto-invoke-services-grpc.md >}})
-- Try out the [hello world quickstart](https://github.com/dapr/quickstarts/blob/master/tutorials/hello-world/README.md) which shows how to use HTTP service invocation or try the samples in the [Dapr SDKs]({{< ref sdks >}})
-- Read the [service invocation API specification]({{< ref service_invocation_api.md >}})
-- Understand the [service invocation performance]({{< ref perf-service-invocation.md >}}) numbers
+### Start using service invocation directly in your app
+Want to skip the quickstarts? Not a problem. You can try out the service invocation building block directly in your application to securely communicate with other services. After [Dapr is installed](https://docs.dapr.io/getting-started), you can begin using the service invocation API in the following ways.
+
+Invoke services using:
+- **Proxying** (recommended set up method)
+  - *HTTP Proxying* - Allows you to just add the `dapr-app-id` header and you're ready to get started. Read more on this here, [Invoke Services using HTTP.]({{< ref howto-invoke-discover-services.md >}}) 
```

Comment:
Looks like you dropped the proxying in favor of just saying HTTP. This doesn't match the wording for HTTP service invocation and gRPC service invocation we adjusted on lines 46-49. Is this intentional? 

I think it would work to either: 
1. Drop the "service invocation" wording after HTTP on line 47 and after gRPC after line 48 or
2. Keep the "service invocation" wording for both lines. 

However whatever decision we converge on, I think it looks better to match the wording with lines 134 and line 153 where we get users to try the API directly in their app. Making sure both sections have wording parity for the same concept might make it easier to conceptualize and associate them as the same exact topics. 

Personally I think dropping the service invocation wording altogether within the nested bullet points and just mentioning it in the main bullet (i.e., "HTTP and gRPC service invocation") looks cleaner but I'm fine with either. So lmk what your preference and thoughts are.

Reply:
This was resolved below.